cognite/neat/graph/extractors/_iodd.py

```diff
@@ -104,29 +151,59 @@ def _text_elements2triples(cls, et_root: Element, namespace: Namespace) -> list[
                     text_id = namespace[id]
 
                     # Create Text node
-                    triples.append((text_id, RDF.type, as_neat_compliant_uri(IODD["Text"])))
+                    triples.append((text_id, RDF.type, as_neat_compliant_uri(IODD["TextObject"])))
```

Comment:
```suggestion
                    triples.append((text_id, RDF.type, IODD.TextObject))
```

Reply:
no need for `as_neat_compliant_uri` since you are not using non-web safe characters